force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -294,22 +416,45 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Person {
         // Declare private two instance variables - (TEXT) name and (Number) age
+        private String name;
+        private Integer age;
         
         // Constructor to initialize the name and age
-    
+        public Person(String name, Integer age) {
+            this.name = name;
+            this.age = age;
+        }
+        
         // Method introduceYourself that returns the details of the person        
         // Return a string in the format "Hello, my name is <name> and I am <age> years old."
         // For example, introduceYourself() should return a string like "Hello, my name is John Doe and I am 28 years old."
+        public String introduceYourself() {  
+            return 'Hello, my name is ' + this.name + ' and I am ' + this.age + ' years old.';
+        }
 
         // Getter method that returns the name of the person
+        public String getName() {
+            return name;
```

Comment:
A general practice for class properties is to use this. keyword to differentiate it. It's not specifically required in apex and this situation but if you had an additional name value then it wouldn't get the correct one. 
```suggestion
            return this.name;
```